packages/web/src/scss/components/Modal/index.html

```diff
@@ -309,7 +309,152 @@ <h2 id="example_3_title" class="ModalHeader__title">Modal with a Form</h2>
     </dialog>
     <!-- Modal: end -->
 
-  </section>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Stacked Modal</h2>
```

Comment:
Why this example? It should be in a separate `Docs` commit as it demonstrates something not related to this PR.

Reply:
done